clouds/clouds-azure/src/test/java/azure/armTemplates/di/suitetests/AzEapHostnameTests.java

```diff
@@ -0,0 +1,49 @@
+package azure.armTemplates.di.suitetests;
+
+
+import azure.core.identification.AzureVirtualMachine;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import sunstone.api.Parameter;
+import sunstone.api.WithAzureArmTemplate;
+import sunstone.api.inject.Hostname;
+
+import static azure.armTemplates.AzureTestConstants.IMAGE_REF;
+import static azure.armTemplates.di.SunstoneResourceAzSuiteTests.group;
+import static azure.armTemplates.AzureTestConstants.instanceName;
+import static org.assertj.core.api.Assertions.assertThat;
+
+
+@WithAzureArmTemplate(parameters = {
+        @Parameter(k = "virtualMachineName", v = instanceName),
+        @Parameter(k = "imageRefId", v = IMAGE_REF)
+},
+        template = "azure/armTemplates/eap.json", group = group, perSuite = true)
+public class AzEapHostnameTests {
+
+    @AzureVirtualMachine(name = instanceName)
```

Comment:
Could you proide default value for `name` parameter  value `azure.armTemplates.AzureTestConstants.instanceName`? So user can write  just

```
  @AzureVirtualMachine
  Hostname eapHostname;
```

Reply:
We have talked this is not possible because instance name must be taken from Azure template.